doc/nrf/known_issues.rst

```diff
@@ -391,6 +391,10 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
+.. rst-class:: v1-8-0
+
+KRKNWK-12301: Packets are retransmitted too early when ACK not received.
```

Comment:
This is not the issue. The issue is that if the transmission fails, the joiner logic fails and the device requires cold reboot.